### ISSUE TITLE
Jane's Fleet Command InstallShield doesn't run

### ISSUE THREAD

**Issue by BEENNath58** [leecher1337/ntvdmx64#183]
Jane's Fleet Command InstallShield looks to be different from other ones. First of all it complains that the app can't run on my PC. This was different from what I would get with a InstallShield having a 16-bit stub. So then I thought it must be 16-bit file only, but OTVDM fails to execute it too. Then I went to dosbox knowing that IS is Windows app and it says that it isn't a DOS executable. 

So Plan B,  Windows XP. Well that one didn't even recognise it as a valid Win32 executable. So only to check the CD wasn't an issue, I went to Windows 98 and surprisingly it launched and installed there. 

So could you look into what's going on with this InstallShield? I assume the PE header is different for this file, I saw some games (one of them was Agent Armstrong)  that behaved such.

**Comment by leecher1337** [comment]
Where to download?

**Comment by BEENNath58** [comment]
> Where to download?

You can get such games in old-games ru

**Comment by leecher1337** [comment]
Not really, did you even try?
https://www.old-games.com/getfile/10185

**Comment by BEENNath58** [comment]
> Not really, did you even try? https://www.old-games.com/getfile/10185

Umm i said old-games ru, not old-games com

**Comment by leecher1337** [comment]
Ah, sorry, I mixed them up :-)

**Comment by BEENNath58** [comment]
Sorry if I wasted your time but it seems like the problem wasn't related to 16-bit stuff at all. I posted the problem in VOGONS in the meantime and one guy figured out Electronic Arts shipped a corrupted InstallShield installer. According to him it remained unreported till Windows 2000 was released because Windows NT "checks the file header more closely than 9x". So when it was fixed they also released another set of InstallShield setup files, which used to install all the updates.

His help helped to regain access to the original 1999 setup.exe that installs the base version of the game. Anyways thanks for offering the support again.
